foundation/foundation-mda/src/main/resources/templates/inference/inference.api.grpc.base.py.vm

```diff
@@ -1,5 +1,5 @@
 from google.protobuf import json_format
-from pydantic.error_wrappers import ValidationError
+from pydantic import ValidationError
```

Comment:
I(informational): in V2, pydantic.error_wrappers packages are moved to pydantic.